content/en/docs/instrumentation/java/_index.md

```diff
@@ -11,7 +11,8 @@ OpenTelemetry Java consists of the following repositories:
 
 - [opentelemetry-java](https://github.com/open-telemetry/opentelemetry-java):
   Components for manual instrumentation including API and SDK as well as
-  extensions, the OpenTracing shim and examples.
+  extensions, the OpenTracing shim.
+- [opentelemetry-java-docs][]: Manual instrumentation examples.
```

Comment:
Note the addition of `opentelemetry-java-docs` here @open-telemetry/java-approvers @open-telemetry/java-instrumentation-approvers.